configure.ac

```diff
@@ -1217,7 +1217,7 @@ AC_SUBST([jsonlib])
 
 PAC_PUSH_ALL_FLAGS()
 PAC_RESET_ALL_FLAGS()
-PAC_CONFIG_SUBDIR_ARGS([modules/json-c],[--disable-werror],[],[AC_MSG_ERROR(json-c configure failed)])
+PAC_CONFIG_SUBDIR_ARGS([modules/json-c],[CFLAGS=-fvisibility=hidden --disable-werror],[],[AC_MSG_ERROR(json-c configure failed)])
```

Comment:
Should we use `VISIBILITY_CFLAGS`? It is set by `PAC_CHECK_VISIBILITY`, if supported.

Reply:
Sure.

Alternatively we could just add commit to the `json-c` submodule. Since this need go along with the `convenience` libtool patch, it might make more sense there.

Reply:
> Alternatively we could just add commit to the `json-c` submodule. Since this need go along with the `convenience` libtool patch, it might make more sense there. What do you think?

Yes, that's a good point. Putting this is `json-c` makes sense.